x-pack/plugins/cloud_security_posture/public/pages/findings/types.ts

```diff
@@ -14,11 +16,18 @@ export interface FindingsBaseURLQuery {
   filters: Filter[];
 }
 
+export interface FindingsBaseProps {
+  dataView: DataView;
+  pitIdRef: MutableRefObject<string>;
+  setPitId(newPitId: string): void;
+}
+
 export interface FindingsBaseEsQuery {
-  index: string;
   query?: {
     bool: BoolQuery;
   };
+  pitIdRef: MutableRefObject<string>;
+  setPitId(newPit: string): void;
```

Comment:
IMO `FindingsBaseEsQuery` shouldn't include properties that are not valid for an ES request, like `pitRef` (should be the string) and `setPitId` (should not be part of this interface as it's a function)